lua/terrortown/menus/gamemode/equipment/base_equipment.lua

```diff
@@ -186,7 +186,9 @@ function CLGAMEMODESUBMENU:Populate(parent)
     end
 
     -- now add custom equipment settings
-    equipment:AddToSettingsMenu(parent)
+    if isfunction(equipment.AddToSettingsMenu) then
+        equipment:AddToSettingsMenu(parent)
+    end
```

Comment:
maybe we could have an else-block with a message/warning? That way this problem is at least discoverable.

Reply:
As a `Dev(1, ....)`. Could do that! But a report in the addon checker that this addon is using the wrong base, would solve the issue as well. We could still do both though

Reply:
yes but the addonchecker is manually curated and we would need something to realize that an equipment is setting the wrong base so that we can put it onto the addonchecker list.
A Dev(1) message would allow us to do so.

Reply:
yes, but my plan was to add an automatic check to the addonchecker, that checks the base of all weapons loaded. So you get the error on startup, not just when opening the UI. And the error would actually be more usable I think

Reply:
tbh i'd rather have a Dev(1) message now.

I'm not sure about the addonchecker addition, but that is a discussion for another time. And we can always have both or replace this once it's added to the addonchecker